templates/zerver/emails/email.css

```diff
@@ -364,57 +246,4 @@ a.button:hover {
         border-radius: 0 !important;
         border-right-width: 0 !important;
     }
-
-    table[class="body"] .btn table {
-        width: 100% !important;
-    }
-
-    table[class="body"] .btn a {
-        width: 100% !important;
-    }
-
-    table[class="body"] .img-responsive {
-        height: auto !important;
-        max-width: 100% !important;
-        width: auto !important;
-    }
-}
-
-@media all {
-    .ExternalClass {
-        width: 100%;
-    }
-
-    .ExternalClass,
-    .ExternalClass p,
-    .ExternalClass span,
-    .ExternalClass font,
-    .ExternalClass td,
-    .ExternalClass div {
-        line-height: 100%;
-    }
```

Comment:
These definitely need a comment, but I'm pretty sure this is a workaround for this Outlook Web Email issue: https://www.htmlemailcheck.com/knowledge-base/recommended-externalclass-css-fix-outlook-com/